pages/establishment/dashboard/views/index.jsx

```diff
@@ -30,33 +30,49 @@ const Index = ({
     pelh,
     nprc
   },
-  allowedActions
-}) => (
-  <Fragment>
-    <Header title={name} />
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-two-thirds">
-        <PanelList
-          panels={links.filter(link => allowedActions.includes(link.permissions)).map(link => <DashboardLink key={link.path} { ...link } />)}
-        />
+  allowedActions,
+  profile: {
+    asruAdmin
+  }
+}) => {
+
+  return (
+    <Fragment>
+      <Header title={name} />
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <PanelList
+            panels={links.filter(link => allowedActions.includes(link.permissions)).map(link => <DashboardLink key={link.path} { ...link } />)}
+          />
+        </div>
+        <Sidebar>
+          <dl>
+            <dt><Snippet>establishmentLicenceNumber</Snippet></dt>
+            <dd>{ licenceNumber }</dd>
+
+            {
+              pelh && <ProfileLink type="pelh" profile={pelh} />
+            }
+            {
+              nprc && <ProfileLink type="nprc" profile={nprc} />
+            }
+          </dl>
+        </Sidebar>
+        <Sidebar>
+          <dl>
```

Comment:
Not sure there should be two sidebars and two `<dl>`s here. Can we remove the 4 lines above here?

Reply:
done